Yoast/wordpress-seo
# Post type archive indexables

## Summary

Creates builder and watcher classes for creating and updating post type archive indexables.

This PR can be summarized in the following changelog entry:

* N/A

## Technical decisions
* Added a migration to increase the length of the `object_type` column of the `indexable` table from 16 to 191 characters.

## Test instructions
This PR can be tested by following these steps:

* Make sure you have a presenter for post type archives that takes it's value directly from the indexable.
* Go to a post type archive, the new presenter should have the same values as the old one.
* Go to your search appearance settings. Change them for a post type archive.
* Go to that post type archive, the changed settings should be reflected in the presenter output.

## UI changes

## Documentation

## Quality assurance
